CHANGELOG.md

```diff
@@ -30,6 +30,13 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 0.37.0a5
```

Comment:
This will be moved to another file as per this PR: https://github.com/up42/up42-py/pull/554